spec/components/tailwinds/form/builder_spec.rb

```diff
@@ -51,15 +70,33 @@
   end
 
   describe '#select' do
-    let(:output) do
-      builder.select :role, %i[admin user]
+    context 'default' do
+      let(:output) { builder.select :role, %i[admin user] }
+
+      it do
```

Comment:
maybe its better to add the name for checks as in `spec/components/tailwinds/form/builder_spec.rb`

Reply:
done